src/app.js

```diff
@@ -0,0 +1,82 @@
+import express from 'express';
+import logger from 'morgan';
+import cors from 'cors';
+import { initializeStatusManager } from './status.js';
+import issue from './issue.js'
+import revoke from './revoke.js'
+import generateSeed from './generate.js';
+import verifyAccess from './verifyAuthHeader.js'
+import { getConfig } from './config.js';
+
+const { enableStatusAllocation } = getConfig();
+
+export async function build(opts = {}) {
+
+    if (enableStatusAllocation) await initializeStatusManager()
+    
+    var app = express();
+
+    app.use(logger('dev'));
+    app.use(express.json());
+    app.use(express.urlencoded({ extended: false }));
+    app.use(cors())
+
+    app.get('/', function (req, res, next) {
+        res.send({ message: 'hello' })
```

Comment:
Let's do something like:

```suggestion
        res.send({ message: 'signing-service server status: ok.' })
```

Reply:
Done.